datafusion/functions/src/string/btrim.rs

```diff
@@ -82,7 +82,11 @@ impl ScalarUDFImpl for BTrimFunc {
     }
 
     fn return_type(&self, arg_types: &[DataType]) -> Result<DataType> {
-        utf8_to_str_type(&arg_types[0], "btrim")
+        if arg_types[0] == DataType::Utf8View {
+            Ok(DataType::Utf8View)
```

Comment:
👍 

Also eventually it would also be possible to return `Utf8View` when the input was `Utf8` and save a copy as well